src/setup-java.ts

```diff
@@ -10,7 +11,7 @@ import { JavaInstallerOptions } from './distributions/base-models';
 
 async function run() {
   try {
-    const versions = core.getMultilineInput(constants.INPUT_JAVA_VERSION, { required: true });
+    const versions = core.getMultilineInput(constants.INPUT_JAVA_VERSION);
```

Comment:
I think we need to update documentation since this input is not required anymore.

Reply:
Oops, I see it will be done in scope of another task.